Trivia Maze/src/application/Main.java

```diff
@@ -433,10 +433,8 @@ public static void setCheatAnswer(boolean cheat)
 	
 	public static void setHorizontalArchway(int y, int x, ArchwayStatus status)
 	{
-		if (stage == null || gameMaze == null)
-			return;
 		
-		else if(status == null)
+		if(status == null)
```

Comment:
Should also check if horizontalArchways is null.

Reply:
Those are two separate methods.